builtin/game/misc_s.lua

```diff
@@ -36,11 +36,8 @@ end
 
 
 function core.setting_get_pos(name)
-	local value = core.settings:get(name)
-	if not value then
-		return nil
-	end
-	return core.string_to_pos(value)
+	core.log("deprecated", "Deprecated usage of setting_get_pos, use core.settings:get_pos() instead")
```

Comment:
Question: What's the value of this deprecation warning? Given that this is just an alias, it should be pretty harmless to keep around for basically forever.

Modders will be annoyed by the deprecation warnings, and would have to do something like `core.settings.get_pos and core.settings:get_pos(name) or core.setting_get_pos(name)`, or patch out the deprecation warning at a game / mod level.

Reply:
I will remove it, it will annoy projects like VoxeLibre which support the last two versions of Luanti.